docs/protocol_overview.md

```diff
@@ -0,0 +1,245 @@
+8. Also the role of **block proposer has some authorities** (chat coordination and agenda inclusion) that can't be cryptographically verified if misused (typically censorship).
+9. Thus there exists **'few of the validators' that can be lazy or commit harmful misuses** of their authorities which aren't really byzantine faults or invalid blocks.
+10. Normally this isn't a problem, if the rounds are short and the block proposer changes regularly, but we're not.
+11. We need a special mechanism to **'veto' the block proposer not to waste the long rounds, in the consensus layer**. Thus we introduce a special variation of Tendermint called *Vetomint*.
+12. In Vetomint, validators can veto the current block proposer, but still the round will progress (changing the block proposer) if all the honest validators either vote or veto.
+
+## Simple and Light Node Operation
+
+Simperby strongly encourages each member to run their own node,
+in order to ensure that the network is decentralized and distributed (and so truly self-hosted). In other words, running a Simperby chain is not about using AWS servers by a few people, but using their own laptops to physically run the chain. To realize that, it is important to **keep the node operation as simple and lightweight as possible**.
+
+### Rarely-online nodes
+
+One of the most important condition for a 'simple' and 'lightweight' node operations is how often the node needs to be online.
+If we want to make the validators run nodes on their personal laptops, for example, it is not realistic to assume that they will be online 24/7.
+
+Since Simperby's BFT consensus assumes an partially-synchronous network, rarely-online nodes can be trivially handled because it's no more than one typical case of an asynchrony, if
+
+1. the consensus round is (or has grown to be) long enough to cover all appearances of the nodes.
+2. at least one node is online where there is a network broadcast
+
+The condition 1 is actually turned out to be a tough challenge in the later sections, so keep in mind it.
+
+### On-demand block production
+
+Rarely-online nodes means rarely-progressed blockchain.
+(note that in any BFT algorithm, at least 2/3 of nodes must have participated in the consensus process to produce a block)
+
+This will inevitably compromise the latency or throughput of the blockchain, but it really doesn't matter because
+**Simperby is NOT a general contract platform for serving public users,**
+**but a governance platform for a permissioned set of members**.
+
+Simperby blockchain includes only governance-approved state transitions (there are very [few exceptions](#consensus-leader)).
+Every governance agenda must be approved by a majority vote of the members so it will take a time anyway (days or even weeks).
+Considering that, naturally, Simperby's performance will be bounded by the governance process, not the consensus slowed by lightweight node operation.
```

Comment:
```suggestion
Considering that, naturally, Simperby's performance is bound by the governance process, not the consensus slowed by lightweight node operation.
```

Reply:
Also, for me it seems like slowed by ~ is describing the consensus which is probably not your intention so maybe need to fix later. Cannot come up with a better sentence now.